api/src/test/java/uk/gov/dhsc/htbhf/dwp/DWPIntegrationTests.java

```diff
@@ -173,7 +179,22 @@ private void assertResponseCorrect(ResponseEntity<EligibilityResponse> response,
         assertThat(response.getStatusCode()).isEqualTo(OK);
         EligibilityResponse eligibilityResponse = response.getBody();
         assertThat(eligibilityResponse).isNotNull();
-        assertThat(eligibilityResponse).isEqualTo(expectedResponse);
+        assertEligibilityResponse(expectedResponse, eligibilityResponse);
+    }
+
+    // checks that two eligibility responses are equal whilst ignoring the order of the children
+    private void assertEligibilityResponse(EligibilityResponse expectedResponse, EligibilityResponse eligibilityResponse) {
+        assertThat(eligibilityResponse).isEqualToIgnoringGivenFields(expectedResponse, "children");
+        assertChildren(expectedResponse.getChildren(), eligibilityResponse.getChildren());
+    }
+
+    private void assertChildren(List<ChildDTO> expected, List<ChildDTO> actual) {
```

Comment:
If anyone can thing of a nicer way to assert that two lists are equal ignoring order (in a null safe way) please say.